app/styles/app.sass

```diff
@@ -2905,7 +2905,7 @@ input:checked + .slider-nolabel:before
 
 $login-button-width: 100%
 .auth-main-container
-  height: 100vh
+  min-height: 100vh
```

Comment:
Any reason to change this? This would break vertical centering of the login form

Reply:
The status container was longer with the new data and was breaking the layout in /scans. Hence made it to minHeight. Have checked the login form and since it doesn't have any content below the form, the vertical centering doesn't seem to be affected. Will there be any other side effects apart from that?

Reply:
My bad. It is affecting the centering.. will fix it.

Reply:
Have reverted this change as it is no longer required.